test/infrastructure/docker/.gitignore

```diff
@@ -17,4 +17,4 @@ manager
 # test results
 *.xml
 
-e2e/resources/**
+e2e/resources/**
```

Comment:
new line?

Reply:
I have to add a new line?

Reply:
The file in the repo has a newline at the end. This change removes the newline. You can configure you text editor / IDE to always ensure there's a newline at the end of the file - you might want to look into that (I can help if you need). I would revert this change as it's not related to your PR.